pandas/tests/reshape/test_pivot.py

```diff
@@ -1625,3 +1625,9 @@ def test_isleapyear_deprecate(self):
 
         with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
             assert isleapyear(2004)
+
+    def test_issue_13292(self):
+        # The below shouldn't raise an exception anymore.
+        frame = pd.DataFrame({'foo': [1, 2, 3]})
+        pd.pivot_table(frame, index=['foo'], aggfunc=len, margins=True,
```

Comment:
you don't need the comment, instead change the test name to something useful like: ``test_pivot_margins_name_unicode`` and put a comment with the issue number